pkg/catalog/routes.go

```diff
@@ -654,9 +671,6 @@ func (mc *MeshCatalog) buildOutboundPermissiveModePolicies(srcServices []service
 		destServices = append(destServices, utils.K8sSvcToMeshSvc(k8sService))
 	}
 
-	// For permissive mode build an outbound policy to every service in the mesh except to itself
-	destServices = difference(destServices, srcServices)
```

Comment:
previously in permissive mode we were preventing services from building routes to itself. However to keep consistency with SMI we will allow this going forward.